Add plugin list (#7921)
pytest-dev/pytest: changelog/5105.doc.rst

> Add automatically generated `plugin_list.rst` and a workflow running on a cronjob for updates.

Reviewer: ```suggestion
Add automatically generated `plugin_list.rst`. The list is updated on a periodic schedule.
```
